docs/source/explore/get-started/install.rst

```diff
@@ -2,39 +2,61 @@
 Install Vircadia
 ################
 
-Vircadia has two different installers. The *Client Installer* comes with everything you need to view and interact with Vircadia's content and users. However, you are unable to host content using this installer. The *Client + Sandbox Installer* has everything the client installer does, but it also enables you to :doc:`host your own content <../../host>` and share it with the world. 
+Vircadia has two different installers. The *Client Installer* comes with everything you need to view and interact with Vircadia's content and users. However, you are unable to host content using this installer. The *Client + Server Installer* has everything the client installer does, but it also enables you to :doc:`host your own content <../../host>` and share it with the world. *The server sandbox is currently only supported on Windows. Alternatively, you can install a server on Linux.*
```

Comment:
```suggestion
Vircadia has two different installers. The *Client Installer* comes with everything you need to view and interact with Vircadia's content and users. However, you are unable to host content using this installer. The *Client + Server Installer* has everything the client installer does, but it also enables you to :doc:`host your own content <../../host>` and share it with the world. *The server installer is currently only provided for Windows. Alternatively, you can manuall install a server on Linux.*
```

Reply:
I see that you want to get rid of "sandbox" but still have the windows and linux version differ, but we cannot really call a proper package a "manual" install, when it is probably less manual than the windows install.

Reply:
Maybe something like
```
*The standalone server is currently only supported on Windows. Alternatively, you can install a dedicated server on Linux.*
```
would work?
On Linux, automatic startscripts are installed on both the ubuntu package and when using vircadia-builder.
I don't know how the windows server looks without the tray icon, but I assume it is just a cmd window without any set up services?

Reply:
Perhaps something along the lines of...
```
*A Client + Server Installer download is only provided for Windows. For Linux you can ... build and deploy a domain server ... <link to docs page describing>*
```

Reply:
I cannot make documentation links italic, but here we go: 546d9b3